packages/nhost-js/tests/graphql.test.ts

```diff
@@ -125,4 +128,55 @@ describe('main tests', () => {
     expect(error).toBeTruthy()
     expect(data).toBeNull()
   })
+
+  it('GraphQL client works with TypedDocumentNode', async () => {
```

Comment:
If we expect to get an error for this response, should we have a different title for the test?